components/rpill/common/rpill.cc

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/rpill/common/rpill.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base/barrier_closure.h"
+#include "base/callback.h"
+#include "base/callback_helpers.h"
+#include "base/strings/string_util.h"
+#include "base/system/sys_info.h"
+
+namespace brave_rpill {
+
+namespace {
+
+struct DeviceNameInfo {
+  std::string manufacturer_name;
+  std::string model_name;
+};
+
+std::string BuildDeviceId(DeviceNameInfo* info) {
```

Comment:
style nit: make this a method on the `DeviceNameInfo` struct

Reply:
Moved